src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const DEF_PRICE = 40;
+  const MID_TERM = 3;
+  const MID_TERM_DISCOUNT = 20;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+
+  const basePrice = days * DEF_PRICE;
+
+  if (days >= LONG_TERM) {
+    return basePrice - LONG_TERM_DISCOUNT;
+  } else if (days >= MID_TERM && days < LONG_TERM) {
+    return basePrice - MID_TERM_DISCOUNT;
+  }
```

Comment:
Since you have a `return` inside the `if` block on line 16, you don't need to use `else if` here. Simply use `if` and remove the condition `days < LONG_TERM` as it's unnecessary due to the previous return statement. This will simplify the logic and make the code easier to read.